packages/cosmic-swingset/src/chain-main.js

```diff
@@ -167,7 +167,11 @@ const makePrefixedBridgeStorage = (
   });
 };
 
-export default async function main(progname, args, { env, homedir, agcc }) {
+export default async function main(
+  progname,
+  args,
+  { env, homedir, path = nativePath, agcc },
```

Comment:
I'd love to make this `path` power required and drop the nativePath import, but don't want to include such a [technically] breaking change in this PR.